pkg/innerring/innerring.go

```diff
@@ -418,6 +418,18 @@ func New(ctx context.Context, log *zap.Logger, cfg *config.Config, errChan chan<
 
 	serveMetrics(server, cfg)
 
+	// TODO: remove deprecated option in future releases
+	if cfg.IsSet("fschain_autodeploy") && cfg.IsSet("fschain.disable_autodeploy") &&
```

Comment:
i think any case when both values are set can be reported as an error, this is smth abnormal to me